optics-core/src/Optics/Core/Extra.hs

```diff
@@ -0,0 +1,17 @@
+module Optics.Core.Extra
+  (
+    is
+  )
+  where
+
+import Optics.Internal.Optic
+import Optics.AffineFold
+
+-- | Check to see if this 'AffineFold' matches.
+--
+-- >>> is _Just Nothing
+-- False
+--
+is :: Is k An_AffineFold => Optic' k is s a -> s -> Bool
+is k = not . isn't k
```

Comment:
Can you inline the definition of `isn't` here so that the compiler has less things to do?

Reply:
Done